test/kickstart-templates/kickstart.ks.template

```diff
@@ -28,6 +28,7 @@ part /boot --fstype=xfs --asprimary --size=800
 part pv.01 --grow
 volgroup rhel pv.01
 logvol / --vgname=rhel --fstype=xfs --size=REPLACE_LVM_SYSROOT_SIZE --name=root
+logvol none --thinpool --vgname=rhel --fstype="None" --size=100 --name=thin
```

Comment:
Do we need to update the partition table comment above to reflect the thin volume?